core/src/main/java/org/radargun/config/VmArgs.java

```diff
@@ -28,6 +33,9 @@
    @Property(doc = "Ignore all VM arguments passed to slave and use only those specified here. Default is false.")
    private boolean ignoreDefault = false;
 
+   @Property(doc = "Print all VM arguments. Default is false.")
+   private boolean printFlagsFinal = false;
```

Comment:
Please use Boolean instead of boolean; we need a tri-state logic to set the parameter to enabled/disabled or not set it at all (null).